doc/src/manual/running-external-programs.md

```diff
@@ -215,20 +221,21 @@ easily and safely just examine its interpretation without doing any damage.
 
 ## Pipelines
 
-Shell metacharacters, such as `|`, `&`, and `>`, are not special inside of Julia's backticks:
-unlike in the shell, inside of Julia's backticks, a pipe is always just a pipe:
+Shell metacharacters, such as `|`, `&`, and `>`, need to be quoted (or escaped) inside of Julia's backticks:
```

Comment:
Looks good. @StefanKarpinski do we want to say anything here about why we're enforcing this?